### PR TITLE
Added GitHub search  

### PR DESCRIPTION
**Pull request by ypratham**
The new command will give the GitHub URL of a particular user.

### CODE REVIEW

**Review by cmulay — APPROVED** [review]

File: main.py [hunk 1]
```diff
@@ -209,6 +216,12 @@ async def on_message(message):
       joke = random_joke()
       await message.channel.send(">>> " + joke)
 
+#Condition to search a user in github
+    if msg.startswith('!pdc git search'):
```
Comment (cmulay) [inline main.py:220]: The command should be `!pdc search github` as git and github are very different from each other

File: main.py [hunk 2]
```diff
@@ -36,7 +36,7 @@
...
+     ">>> **Help Commands** \n\nThese are the available commands:\n\n1. `!pdc help` - Dailogue of all commands\n2. `!pdc info` -  Gives info of bot\n3. `!pdc about` -  Returns server information\n4. `!pdc discord` - Provides invitation link for the discord server\n5. `!pdc github` - Provides link to the github organisation\n6. `!pdc core team` - Returns current Core Member\n7. `!pdc list projects` - Returns active projects\n8. `!pdc quote`s - Returns random quote\n9. `!pdc events` - Returns upcoming events\n10. `!pdc new-event` - Add new event\n11. `!pdc delete-event` - Delete an event\n12. `!pdc list-events` - List all events\n13. `!pdc event-syntax` - List all syntax for events command\n14. `!pdc new project` - add new project to the list\n15. `!pdc delete project` - delete a project from the list\n16. `!pdc meme` - Returns meme\n17. `!pdc joke` - Returns a joke\n18. `!pdc git search` - get the github url of a user\n\n _Our bot is Open Source_"
```
Comment (cmulay) [inline main.py:39]: Update the command guidelines as well

**Review by cmulay — CHANGES_REQUESTED** [review]

File: main.py (hunk 1 above)
Comment (ypratham) [reply main.py:220]: Okay, I will change

File: main.py (hunk 2 above)
Comment (ypratham) [reply main.py:39]: Yeah sure

**Review by cmulay — APPROVED** [review]
LGTM !

**Pull request merged**